### PR TITLE
[1.22.0] disaster recovery cli commands

### PR DESCRIPTION
**Pull request by salahalsaleh**

### CODE REVIEW

**Comment by markpundsack** [comment]
I'm curious why `restore` doesn't need a namespace.

**Review by markpundsack — APPROVED** [review]

File: content/kots-cli/restore.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+linktitle: "restore"
+title: kots restore
+weight: 90220
+---
+
+The `kots restore` restores an instance backup for disaster recovery. 
+
+### Usage
+```bash
+kubectl kots restore [flags]
```
Comment (markpundsack) [inline content/kots-cli/restore.md:12]: Technically, one of these flags, `--from-backup` is required, right? As written, it says all flags are optional. Maybe `--from-backup` should be explicitly listed here, not as a "flag".

File: content/kots-cli/backup.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+### Usage
+```bash
+kubectl kots backup [flags]
+```
+* _Provide `[flags]` according to the table below_
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `-h, --help` | |   help for upstream |
+| `-n, --namespace`| string |     the namespace where the admin console is running |
```
Comment (markpundsack) [inline content/kots-cli/backup.md:19]: Is there a default?

File: content/kots-cli/restore.md (hunk 1 above)
Comment (markpundsack) [reply content/kots-cli/restore.md:12]: I should take that back. The [flags] convention doesn't necessarily mean it's optional. Or at least that's how we've been using it elsewhere.

Comment (salahalsaleh) [reply content/kots-cli/restore.md:12]: i updated the flag to mention that it's required. didn't include it in the command itself though.

File: content/kots-cli/backup.md (hunk 2 above)
Comment (salahalsaleh) [reply content/kots-cli/backup.md]: yes, updated.

File: content/kots-cli/backup.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+date: 2019-10-30
+linktitle: "backup"
+title: kots backup
+weight: 90200
+---
+
+The `kots backup` creates a new instance backup for disaster recovery.
```
Comment (markpundsack) [inline content/kots-cli/backup.md:8]:
```suggestion
The `kots backup` creates a new instance snapshot for disaster recovery. 
```

File: content/kots-cli/backup-ls.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+date: 2019-10-30
+linktitle: "backup ls"
+title: kots backup ls
+weight: 90210
+---
+
+The `kots backup ls` shows a list of all the available instance backups for disaster recovery.
```
Comment (markpundsack) [inline content/kots-cli/backup-ls.md:8]:
```suggestion
The `kots backup ls` shows a list of all the available instance snapshots for disaster recovery. 
```

File: content/kots-cli/restore.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+date: 2019-10-30
+linktitle: "restore"
+title: kots restore
+weight: 90220
+---
+
+The `kots restore` restores an instance backup for disaster recovery.
```
Comment (markpundsack) [inline content/kots-cli/restore.md:8]:
```suggestion
The `kots restore` restores an instance snapshot for disaster recovery. 
```

File: content/kots-cli/restore-ls.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+date: 2019-10-30
+linktitle: "restore ls"
+title: kots restore ls
+weight: 90230
+---
+
+The `kots restore ls` shows a list of all the available instance restores for disaster recovery.
```
Comment (markpundsack) [inline content/kots-cli/restore-ls.md:8]: Hmm... not sure what to call these. "list all running and completed restores" or "restorations"? I guess we can go with this as is.

**Review by markpundsack — CHANGES_REQUESTED** [review]

**Pull request merged**